CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 Details changes in each release of EarlGrey. EarlGrey follows [semantic versioning](http://semver.org/).
 
+## [1.15.0](https://github.com/google/EarlGrey/tree/1.15.0) (08/02/2018)
+```
+Baseline: [59ce3b6c]
++ [59ce3b6c]: Fix default Swift version in EarlGreyExampleSwiftTests xcode project
+```
+
+### Enhancements
+* Added support for accessibility in iOS 12.
+* Updated the visibility checker to support keyboards in iOS 12.
+* Updated Analytics Configurations.
+* Fixed Formatting Issues.
+* Updated invalid api and compatibility docs.
+
+### Compatibility
+* EarlGrey has now been tested for working till Xcode version 10.0 beta 2. Any small unit test breakages with Xcode 9.0's  XCTest Exception are being invesigated.
```

Comment:
double space after 9.0's

invesigated => investigated

also what does this sentence really mean?
"Any small unit test breakages with Xcode 9.0's  XCTest Exception are being invesigated"

why are we seeing issues with 9?

Reply:
Yeah, XCTestCase Unit Tests. We need to change the exception name for them.